pkgs/by-name/pd/pd/package.nix

```diff
@@ -0,0 +1,79 @@
+{ rev ? "a0f8550034958d71cb998bfc1d029bb02ff83320"
+, hash ? "sha256-0He3Ryqy4RxYuR2HbwGHJxegfOuKJ91B/JI+z2AK8wk="
+
+, lib
+
+, stdenv_32bit
+, fetchFromGitHub
+, fetchpatch
```

Comment:
```suggestion
```
Unused lambda pattern

Reply:
Also squash 7d818fea60246301f9e4870f847e24ca815fd380 with 60c9c632ded5b048c612c47c7bc753b754ccff4b to make commits like this:
```
maintainers: add PaulGrandperrin
pd: init at 0-unstable-2024-04-25
```

Reply:
I intended to squash everything in my next commit :)